vonage/changelog.d/19012.changed

```diff
@@ -0,0 +1 @@
+Vonage integration updated
```

Comment:
Can you remove this changelog file? It's not necessary for a crawler integration like this.